### PR TITLE
cottontail: new tool

### PR DESCRIPTION
**Pull request by gkroon**
New tool I'm using on an assignment

### CODE REVIEW

File: net-analyzer/cottontail/cottontail-20190426.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+inherit distutils-r1
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/QKaiser/cottontail.git"
+	KEYWORDS=""
+else
+	KEYWORDS="~amd64 ~x86"
+	EGIT_COMMIT="77ed037395a3e98bf5339aa14f6f02c002f3ab51"
+	SRC_URI="https://github.com/QKaiser/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (linxon) [inline net-analyzer/cottontail/cottontail-20190426.ebuild]: you no need use ${PN} in the static place of the URI
use only ${P}/${PV} where is it necessary

```diff
-SRC_URI="https://github.com/QKaiser/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/QKaiser/cottontail/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

File: net-analyzer/cottontail/cottontail-20190426.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+DESCRIPTION="Capture all RabbitMQ messages being sent through a broker"
+HOMEPAGE="https://github.com/QKaiser/cottontail"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
```
Comment (linxon) [inline net-analyzer/cottontail/cottontail-20190426.ebuild]: change it to:
```diff
+DEPEND="${PYTHON_DEPS}"
-RDEPEND="
+RDEPEND="
 	dev-python/coloredlogs[${PYTHON_USEDEP}]
 	dev-python/pika[${PYTHON_USEDEP}]
 	dev-python/requests[${PYTHON_USEDEP}]
 	dev-python/urllib3[${PYTHON_USEDEP}]
 	dev-python/verboselogs[${PYTHON_USEDEP}]
 "
-DEPEND="${RDEPEND}"
```

File: profiles/pentoo/base/package.accept_keywords/net-analyzer [hunk 3]
```diff
@@ -146,3 +146,5 @@ net-analyzer/GyoiThon **
 net-analyzer/dirble-bin
 
 net-analyzer/cameradar
+
+~net/analyzer/cottontail
```
Comment (linxon) [inline profiles/pentoo/base/package.accept_keywords/net-analyzer]: `~net/analyzer/cottontail` -> `net-analyzer/cottontail`

**Comment by gkroon** [comment]
Thanks for reviewing!

**Pull request closed without merging**